packages/arb-token-bridge-ui/src/components/TransferPanel/USDCDeposit/USDCDepositWithArbBridgeInfo.tsx

```diff
@@ -0,0 +1,61 @@
+import { CommonAddress } from '../../../util/CommonAddressUtils'
+import { ExternalLink } from '../../common/ExternalLink'
+import { Checkbox } from '../../common/Checkbox'
+
+export function USDCDepositWithArbBridgeInfo({
+  USDCcheckboxChecked,
+  setUSDCcheckboxChecked
+}: {
+  USDCcheckboxChecked: boolean
+  setUSDCcheckboxChecked: React.Dispatch<React.SetStateAction<boolean>>
+}) {
+  const externalLinkClassnames = 'arb-hover text-blue-link underline'
+
+  return (
+    <div className="flex flex-col space-y-3">
+      <p className="font-light">
+        Receive{' '}
+        <ExternalLink
+          className={externalLinkClassnames}
+          href={`https://arbiscan.io/token/${CommonAddress.ArbitrumOne['USDC.e']}`}
+        >
+          Bridged USDC (USDC.e)
+        </ExternalLink>{' '}
+        on Arbitrum One using Arbitrum’s native bridge.
+      </p>
+
+      <div className="flex flex-col space-y-6">
+        <Checkbox
```

Comment:
What if we only kept this Checkbox in a separate file (e.g. `USDCDepositConfirmationDialogCheckbox` and inlined the rest? Would make the props cleaner, too.